bip78/src/receiver/mod.rs

```diff
@@ -42,23 +53,91 @@ impl UncheckedProposal {
         })
     }
 
+    /// The Sender's Original PSBT
     pub fn get_transaction_to_check_broadcast(&self) -> bitcoin::Transaction {
         self.psbt.clone().extract_tx()
     }
 
-    pub fn assume_broadcastability_was_verified(self) -> UnlockedProposal {
-        UnlockedProposal {
+    /// Receiver MUST check that the Original PSBT from the sender
+    /// can be broadcast, i.e. `testmempoolaccept` bitcoind rpc returns { "allowed": true,.. }
+    /// for `get_transaction_to_check_broadcast()` before calling this method.
+    ///
+    /// Check this if you generate bitcoin uri to receive PayJoin on sender request without manual human approval, like a payment processor.
+    /// Such so called "interactive" receivers are otherwise vulnerable to probing attacks.
+    /// If a sender can make requests at will, they can learn which bitcoin the receiver owns at no cost.
+    /// Broadcasting the Original PSBT after some time in the failure case makes incurs sender cost and prevents probing.
+    ///
+    /// Call this after checking downstream.
+    pub fn attest_tested_and_scheduled_broadcast(self) -> MaybeInputsOwned {
```

Comment:
I previously chose `assume` word because it has similar meaning to `assume` in `MaybeUninit::assume_init` (except it's not `unsafe` but that's obvious from the signature). Can you explain your preference for `attest`?

Reply:
I didn't know there was canon for this already. I've adopted assume for consistency's sake.